Jint.Tests/Runtime/JsonTests.cs

```diff
@@ -48,5 +56,21 @@ public void ShouldReportHelpfulSyntaxErrorForInvalidJson(string json, string exp
             Assert.NotNull(error);
             Assert.Equal("SyntaxError", error.Get("name"));
         }
+
+        [Theory]
+        [InlineData("[[]]", "[\n  []\n]")]
+        [InlineData("[ { a: [{ x: 0 }], b:[]} ]",
+            "[\n  {\n    \"a\": [\n      {\n        \"x\": 0\n      }\n    ],\n    \"b\": []\n  }\n]")]
+        public void ShouldSerializeWithCorrectIndentation(string script, string expectedJson)
+        {
+            var engine = new Engine();
+            engine.SetValue("x", engine.Evaluate(script));
+
+            var result = engine.Evaluate("JSON.stringify(x, null, 2);").AsString();
+
+            _outputHelper.WriteLine(result);
```

Comment:
Probably not needed? Only adds unnecessary output to CI run

Reply:
will remove